README.md

```diff
@@ -23,10 +23,14 @@ The server component to the TourTrak system built using Django.
 * rabbitmq-server
 
 
-## Install PostGIS
+## Install Postgres with PostGIS (Install The Database)
 
-1. Install postgreSQL. Refer to official documentation. After, install postGIS, which is a postgreSQL extention for handling spatial data by doing `apt-get install postgresql-9.1-postgis`
-2. Setup the postgreSQL database:
+<i>(This is where you begin the server setup instructions)</i><br>
+1. Install postgreSQL
+	<br>`sudo apt-get install postgresql`<br>
+2. After, install postGIS, which is a postgreSQL extention for handling spatial data
+	<br>`sudo apt-get install postgresql-9.3-postgis-2.1`<br>
+3. Setup the postgreSQL database:
 
 * switch to the default postgresql user by running `su postgres`
```

Comment:
I'm going to also suggest that you note that you can't use `sudo` while logged in as the postgres user, so after running the database commands, the user needs to switch users.